openvdb/openvdb/tree/LeafManager.h

```diff
@@ -586,7 +586,12 @@ class LeafManager
         using LeafParentT = typename CopyConstness<TreeType, NonConstLeafParentT>::Type;
 
         std::deque<LeafParentT*> leafParents;
-        mTree->getNodes(leafParents);
+        if constexpr(std::is_same<LeafParentT, RootNodeType>::value) {
+            leafParents.emplace_back(&mTree->root());
+        }
+        else {
+            mTree->getNodes(leafParents);
```

Comment:
Maybe getNodes should ust do the above if NodeT* of the provided array is the RootNode